examples/run_example.sh

```diff
@@ -5,19 +5,10 @@
 #   User
 #   Device
```

Comment:
Maybe it's worth to actually include all available examples here?
It's nowhere documented how user can derive the name.

Reply:
The place for that is in readme in "running examples" section.
BTW, I consider this script redundant, because one line needed to run an example is provided in readme.